docker/jenkins2-centos/contrib/jenkins/kube-slave-common.sh

```diff
@@ -155,17 +163,28 @@ function generate_kubernetes_config() {
 # generate_kubernetes_credentials generates the credentials entry for the
 # kubernetes service account.
 function generate_kubernetes_credentials() {
-  echo "<entry>
-      <com.cloudbees.plugins.credentials.domains.Domain>
-        <specifications/>
-      </com.cloudbees.plugins.credentials.domains.Domain>
-      <java.util.concurrent.CopyOnWriteArrayList>
-        <org.csanchez.jenkins.plugins.kubernetes.ServiceAccountCredential plugin=\"kubernetes@0.4.1\">
+  echo "<org.csanchez.jenkins.plugins.kubernetes.ServiceAccountCredential plugin=\"kubernetes@0.8\">
           <scope>GLOBAL</scope>
-          <id>1a12dfa4-7fc5-47a7-aa17-cc56572a41c7</id>
+          <id>aerogear-digger-jenkins-kubernetes-service-account-credential</id>
           <description></description>
         </org.csanchez.jenkins.plugins.kubernetes.ServiceAccountCredential>
-      </java.util.concurrent.CopyOnWriteArrayList>
-    </entry>
-    "
+       "
 }
+
+# generate_osx_slave_credentials generates the credentials entry for the
+# OSX slave.
+function generate_osx_slave_credentials() {
```

Comment:
Can we move this source code somewhere else, just in case we would remove this file?